charts/actions-runner-controller/crds/actions.summerwind.dev_horizontalrunnerautoscalers.yaml

```diff
@@ -1,8 +1,9 @@
+---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.7.0
+    controller-gen.kubebuilder.io/version: v0.11.3
```

Comment:
Do we have to do this upgrade in this PR?

Reply:
Doesn't it require an upgrade of codegen or something?

Reply:
I guess not really. It just used this version to generate CRD that will be fed to the API server
As long as the API server understands the CRD and serves those objects to the controller, we should be okay
And it is just an annotation, so I guess it just provides more information of how this CRD is generated. It shouldn't influence the execution